src/server/func_services/func_server.js

```diff
@@ -316,23 +321,19 @@ function check_event_permission(req) {
     }
 }
 
-function _get_func_code_stream(req, func_code) {
+// _get_func_code will return the code in base64, the code size and it's sha256
+async function _get_func_code_b64(req, func_code) {
+    let code;
+    let code_size;
+    const sha256 = crypto.createHash('sha256');
     if (func_code.zipfile_b64) {
         // zipfile is given as base64 string
-        return new stream.Readable({
-            read(size) {
-                this.push(Buffer.from(func_code.zipfile_b64, 'base64'));
-                this.push(null);
-            }
-        });
+        code = func_code.zipfile_b64;
+        code_size = Buffer.from(func_code.zipfile_b64).length;
```

Comment:
Is there a reason you did not do 'base64' or you just missed it?
Buffer.from(func_code.zipfile_b64, 'base64')

Reply:
This flow is when the zipfile is given as base64 string